### PR TITLE
[BENCHMARK_APP] Help message cleanup

### PR DESCRIPTION
**Pull request by tomdol**
### Details:
 - Cleanup of some messages
 - Vertical alignment of help options, adjustment of missing whitespaces
 - Unification of quotes

### CODE REVIEW

**Review by rkazants — COMMENTED** [review]
You changed third-party dependency here: [src/plugins/intel_cpu/thirdparty/onednn](https://github.com/openvinotoolkit/openvino/pull/14786/files#diff-ca2918c90dabbfc4f712c4a3cd6a6fb3b50b47805c94a43dcea21054f2ad7db7)
Please check:)

**Review by rkazants — CHANGES_REQUESTED** [review]
third-party component is changed by mistake. Please correct.

File: samples/cpp/benchmark_app/benchmark_app.hpp [hunk 1]
```diff
@@ -226,7 +226,9 @@ static const char load_from_file_message[] = "Optional. Loads model from file di
                                              " All CNNNetwork options (like re-shape) will be ignored";
 
 // @brief message for inference_precision
-static const char inference_precision_message[] = "Optional. Inference precision";
+static const char inference_precision_message[] =
+    "Optional. Specifies the inference precision. Example #1: '-infer_precision bf16'. Example #2: '-infer_precision "
+    "CPU:bf16,GPU:f32'";
```
Comment (Wovchena) [inline samples/cpp/benchmark_app/benchmark_app.hpp:231]: Please align this with Python benchmakr_app

File: samples/cpp/benchmark_app/benchmark_app.hpp [hunk 2]
```diff
@@ -121,7 +121,7 @@ static const char batch_size_message[] =
 
 // @brief message for CPU threads pinning option
 static const char infer_threads_pinning_message[] =
-    "Optional. Explicit inference threads binding options (leave empty to let the OpenVINO to make a choice):\n"
+    "Optional. Explicit inference threads binding options (leave empty to let the OpenVINO make a choice):\n"
```
Comment (Wovchena) [inline samples/cpp/benchmark_app/benchmark_app.hpp:124]: Please update https://github.com/openvinotoolkit/openvino/tree/master/samples/cpp/benchmark_app#-all-configuration-options

**Review by Wovchena — CHANGES_REQUESTED** [review]

**Comment by PiotrKrzem** [comment]
`git checkout origin/master -- src/plugins/intel_cpu/thirdparty/onednn` to fix the submodule, if you are wondering :)

File: samples/cpp/benchmark_app/benchmark_app.hpp [hunk 3]
```diff
@@ -402,48 +404,48 @@ static void show_usage() {
...
     std::cout << std::endl;
-    std::cout << "    -h, --help                " << help_message << std::endl;
-    std::cout << "    -m \"<path>\"               " << model_message << std::endl;
-    std::cout << "    -i \"<path>\"               " << input_message << std::endl;
-    std::cout << "    -d \"<device>\"             " << target_device_message << std::endl;
-    std::cout << "    -extensions \"<absolute_path>\" " << custom_extensions_library_message << std::endl;
-    std::cout << "    -c \"<absolute_path>\"      " << custom_cldnn_message << std::endl;
-    std::cout << "    -hint \"performance hint (latency or throughput or cumulative_throughput or none)\"   "
+    std::cout << "    -h, --help                    " << help_message << std::endl;
+    std::cout << "    -m  <path>                    " << model_message << std::endl;
```
Comment (Wovchena) [inline samples/cpp/benchmark_app/benchmark_app.hpp:408]: You should remove quotes from README as well.
Just run benchmark_app with -h and paste the result

File: samples/cpp/benchmark_app/benchmark_app.hpp [hunk 4]
```diff
@@ -402,48 +404,48 @@ static void show_usage() {
...
+    std::cout << "    -i  <path>                    " << input_message << std::endl;
```
Comment (Wovchena) [inline samples/cpp/benchmark_app/benchmark_app.hpp:409]: Just curious, why you put two spaces between `i` and `<path>`?

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: samples/cpp/benchmark_app/benchmark_app.hpp (hunk 4 above)
Comment (tomdol) [reply samples/cpp/benchmark_app/benchmark_app.hpp:409]: This is just for vertical alignment between nearby lines. It doesn't mean that 2 spaces are actually required.

File: samples/cpp/benchmark_app/benchmark_app.hpp (hunk 3 above)
Comment (tomdol) [reply samples/cpp/benchmark_app/benchmark_app.hpp:408]: Got it, I've updated the whole thing now.

**Review by Wovchena — APPROVED** [review]

**Review by PiotrKrzem — APPROVED** [review]

**Pull request merged**